src/index.html

```diff
@@ -20,6 +20,7 @@
     <link rel="icon" type="image/png" sizes="96x96" href="/static/favicons/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/static/favicons/favicon-16x16.png">
     <link rel="manifest" href="/static/manifest.json">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/twbs/bootstrap/v4-dev/dist/css/bootstrap.css">
```

Comment:
Please just custom style the button and not use bootstrap, this is the wrong way to add CSS for this project, you would do it by installed an NPM module but we don't need to install all of bootstrap the purpose for a button.